src/config/index.ts

```diff
@@ -193,6 +193,13 @@ export const config = convict({
     format: String,
     default: 'staging',
     env: 'STAGING_PREFIX'
+  },
+
+  announcementContent: {
+    doc: 'A user-visible global announcement to inform the the user. Set no empty to hide.',
```

Comment:
Might be worth a rename to match ['Service status' pattern](https://github.com/alphagov/govuk-design-system-backlog/issues/266)

Got a little typo too

```suggestion
  serviceStatus: {
    doc: 'A user-visible global announcement to inform the the user. Set to empty to hide.',
```